roles/gluster_brick/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for gluster_brick
+
+gluster_cluster_volume: glustervol
```

Comment:
For add-brick let us not assume a default volume name. We might end up adding a brick to an unintended volume.